spec/requests/articles/articles_show_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe "ArticlesShow", type: :request do
+  let_it_be(:user) { create(:user) }
+  let_it_be(:organization) { create(:organization) }
+  let_it_be(:organization_article) { create(:article, organization_id: organization.id) }
+  let_it_be(:article, reload: true) { create(:article, user: user, published: true) }
+
+  describe "GET /:slug (articles)" do
+    before do
+      get article.path
+    end
+
+    it "returns a 200 status when navigating to the article's page" do
+      expect(response).to have_http_status(:success)
+    end
+
+    it "renders the proper title" do
+      expect(response.body).to include CGI.escapeHTML(article.title)
+    end
+
+    it "renders the proper published at date" do
+      expect(response.body).to include CGI.escapeHTML(article.readable_publish_date)
+    end
+
+    it "renders the proper modified at date" do
+      time_now = Time.current
+      article.edited_at = time_now
+      expect(response.body).to include CGI.escapeHTML(article.readable_edit_date)
```

Comment:
same question about this one, if i'm missing some context here, again, please do let me know!

Reply:
I've since changed this to make it more reviewer/future programmer-friendly. Let me know what you think. 😄